packages/@react-spectrum/s2/src/Radio.tsx

```diff
@@ -74,7 +74,9 @@ const circle = style<RenderProps>({
   display: 'flex',
   alignItems: 'center',
   justifyContent: 'center',
-  transition: 'default', /* RSP v3 was 'all 200ms' */
+  transition: '[border-width]',
```

Comment:
i've updated the animation for radio's to more closely resemble what we had in V3